x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/search/SearchApplicationIndexServiceTests.java

```diff
@@ -69,7 +69,7 @@ public void testEmptyState() throws Exception {
     }
 
     public void testCreateSearchApplication() throws Exception {
-        final SearchApplication searchApp = new SearchApplication("my_search_app", new String[] { "index_1" }, null);
+        final SearchApplication searchApp = new SearchApplication("my_search_app", new String[] { "index_1" }, null, null);
```

Comment:
I think we'll want some tests where templates are specified

Reply:
I think we have that in x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/search/SearchApplicationTests.java?

Reply:
Hmm OK, if we're going to do that, maybe at least add the template field to the data in `testMerge()` in `SearchApplicationTests.java`, and in this file sending in a String rather than null in `testUpdateSearchApplication()`?

Reply:
done - updated `testUpdateSearchApplication` - also updated `randomSearchApplication` to use a random `SearchTemplate`.